### PR TITLE
Bump up gRPC version, and use its new release repository

### PR DESCRIPTION
**Pull request by alanlvle**
update:
1.grpc update to @grpc/grpc-js
2.update Authentication Method to realize

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +70,6 @@
   },
   "dependencies": {
     "google-protobuf": "^3.14.0",
-    "grpc": "^1.10.1",
```
Comment (wu-sheng) [inline package.json:72]: No gRPC dependency?

File: scripts/protoc.sh [hunk 2]
```diff
@@ -24,17 +24,16 @@ mkdir -p $OUT_DIR || true
 cd "${ROOT_DIR}"/protocol || exit
 
 PROTOC_GEN_TS_PATH="${ROOT_DIR}/node_modules/.bin/protoc-gen-ts"
-PROTOC_PLUGIN="${ROOT_DIR}/node_modules/.bin/grpc_tools_node_protoc_plugin"
+#PROTOC_PLUGIN="${ROOT_DIR}/node_modules/.bin/grpc_tools_node_protoc_plugin"
```
Comment (wu-sheng) [inline scripts/protoc.sh:27]: Why comment out? Rather than deleting?

File: src/agent/protocol/grpc/AuthInterceptor.ts [hunk 3]
```diff
@@ -17,23 +17,32 @@
...
+//       // tslint:disable-next-line:ban-types
+//       start(metadata: Metadata, listener: Listener, next: Function) {
+//         if (config.authorization) {
+//           metadata.add('Authentication', config.authorization);
+//         }
+//         next(metadata, listener);
+//       }
+//     })(),
+//   );
+// }
```
Comment (wu-sheng) [inline src/agent/protocol/grpc/AuthInterceptor.ts:40]: Same question.

**Comment by wu-sheng** [comment]
A lot of format changes, I can't see they are expected, please make sure you submitted the necessary changes.

File: package.json (hunk 1 above)
Comment (alanlvle) [reply package.json:72]: use "@grpc/grpc-js": "^1.4.4"

Comment (wu-sheng) [reply package.json:72]: It is in `devDependencies` only

File: scripts/protoc.sh (hunk 2 above)
Comment (alanlvle) [reply scripts/protoc.sh:27]: new grpc not use grpc_tools_node_protoc_plugin.
https://www.npmjs.com/package/grpc_tools_node_protoc_ts

Comment (wu-sheng) [reply scripts/protoc.sh:27]: This is not the point. The key is removing useless codes, not comment them out.

Comment (alanlvle) [reply scripts/protoc.sh:27]: ok

File: package.json (hunk 1 above)
Comment (alanlvle) [reply package.json:72]: ok

File: src/agent/protocol/grpc/clients/TraceReportClient.ts [hunk 4]
```diff
@@ -53,14 +52,14 @@ export default class TraceReportClient implements Client {
...
-      emitter.emit('segments-sent');  // reset limiter in SpanContext
+      emitter.emit('segments-sent'); // reset limiter in SpanContext
 
       try {
         if (this.buffer.length === 0) {
           return;
         }
 
-        const stream = this.reporterClient.collect((error, _) => {
+        const stream = this.reporterClient.collect(AuthInterceptor(),(error, _) => {
```
Comment (kezhenxu94) [inline src/agent/protocol/grpc/clients/TraceReportClient.ts:62]: Please add a space after `,`

File: src/agent/protocol/grpc/AuthInterceptor.ts [hunk 5]
```diff
@@ -17,23 +17,14 @@
...
-        if (config.authorization) {
-          metadata.add('Authentication', config.authorization);
-        }
-        next(metadata, listener);
-      }
-    })(),
-  );
+export default function AuthInterceptor() { 
+  const mata = new grpc.Metadata()
+  if(config.authorization){
```
Comment (kezhenxu94) [inline src/agent/protocol/grpc/AuthInterceptor.ts:26]: Please add a space before `(`, after `)`

Comment (alanlvle) [reply src/agent/protocol/grpc/AuthInterceptor.ts]: update 3da6ed9155da8a0aebb36569956adae4bf382a02

**Comment by kezhenxu94** [comment]
@alanlvle can you please also update the license information in this directory https://github.com/apache/skywalking-nodejs/tree/master/dist ? Others look good to me, thank you!

**Comment by alanlvle** [comment]
@kezhenxu94 Yes, what are the changes?

**Comment by wu-sheng** [comment]
Generally this part, https://github.com/apache/skywalking-nodejs/blob/master/dist/LICENSE#L211. You need to update the LICENSE file according to the latest dependencies, even dependencies of dependencies, which is required by the ASF and Apache 2.0 License.

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**